src/interpreter/mod.rs

```diff
@@ -118,8 +118,8 @@ enum CachedMir<'mir, 'tcx: 'mir> {
 
 /// Represents the action to be taken in the main loop as a result of executing a terminator.
 enum TerminatorTarget {
-    /// Make a local jump to the given block.
-    Block(mir::BasicBlock),
+    /// Make a local jump to the next block
+    Block,
```

Comment:
So you were just making `Block` more similar to `Call`? It's not actually required by the rest of the PR?

Just wondering. I don't mind the PR, and I'll probably look for other ways to refactor `TerminatorTarget`.